openstack-mixin/config.libsonnet

```diff
@@ -0,0 +1,29 @@
+{
+  filteringSelector: 'job="integrations/openstack"',
+  groupLabels: ['job'],
+  instanceLabels: ['instance'],
+
+  uid: 'openstack',
+  dashboardTags: [self.uid],
+  dashboardPeriod: 'now-30m',
+  dashboardTimezone: 'default',
+  dashboardRefresh: '1m',
+
+  alertsWarningPlacementHighMemoryUsage: 80,  // %
+  alertsCriticalPlacementHighMemoryUsage: 90,  // %
+  alertsWarningNovaHighVMMemoryUsage: 80,  // %
+  alertsWarningNovaHighVMVCPUUsage: 80,  // %
+  alertsCriticalNeutronHighDisconnectedPortRate: 25,  // %
+  alertsCriticalNeutronHighInactiveRouterRate: 15,  // %
+  alertsWarningCinderHighBackupMemoryUsage: 80,  // %
+  alertsWarningCinderHighVolumeMemoryUsage: 80,  // %
+  alertsWarningCinderHighPoolCapacityUsage: 80,  // %
+
+  // logs lib related
+  enableLokiLogs: true,
+  logsExtraFilters: '',
+  extraLogLabels: ['level', 'service'],
+  logsVolumeGroupBy: 'level',
+  logsFilteringSelector: self.filteringSelector,
```

Comment:
Missed this in the first review, but as per usual the logs dashboard is missing the required regex to select the correct datasource by default. Currently it defaults to alert-state-history
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/d8c80589-2f76-4aad-bd14-9cac5068e343)